npm-package-update/npm-update.yaml

```
@@ -0,0 +1,24 @@
+name: npm-update-package.json
+description: Use NPM to update dependencies in package.json.
+
+# NPM Update.
+
+on:
+  - repositoriesMatchingQuery: patternType:regexp file:package.json repo:sourcegraph-testing/docker-image-pin$
+
+# In each repository
+steps:
+  - run: |
+      apk add --update nodejs npm 
+      npm install -g npm-check-updates 
+      ncu -u
+    container: alpine:3
```

This could potentially be simplified further by using an [`alpine` tagged `node` image](https://hub.docker.com/_/node), which would avoid the need to install anything from `apk` and would allow the user to provide a specific version of Node:

```suggestion
  - run: |
      npm install -g npm-check-updates 
      ncu -u
    container: node:16-alpine
```

---

The amount of time I spent failing to google this correctly before giving up. Thank you!!